head/src/main/kotlin/io/qalipsis/core/head/jdbc/repository/DataSeriesRepository.kt

```diff
@@ -58,17 +58,48 @@ internal interface DataSeriesRepository : CoroutineCrudRepository<DataSeriesEnti
     suspend fun findByTenantAndReference(tenant: String, reference: String): DataSeriesEntity
 
     @Query(
-        """SELECT *
+        """SELECT id, reference, version, tenant_id, display_name, data_type,value_name,
+            color, filters, field_name, aggregation_operation, timeframe_unit_ms, display_format,
+            query, color_opacity,
+            CASE WHEN data_series.tenant_id <> (SELECT id FROM tenant WHERE reference = :tenant) THEN 'READONLY'
```

Comment:
This will generate a lot of subqueries, perhaps a LEFT JOIN ON tenant would be better here to select the tenant only once.

Same remark for all the select (not count) queries.